packages/components/src/tools-panel/tools-panel-header/component.tsx

```diff
@@ -11,9 +17,15 @@ import DropdownMenu from '../../dropdown-menu';
 import MenuGroup from '../../menu-group';
 import MenuItem from '../../menu-item';
 import { useToolsPanelHeader } from './hook';
-import { contextConnect } from '../../ui/context';
+import { contextConnect, WordPressComponentProps } from '../../ui/context';
+import type {
+	ToolsPanelControlsGroupProps,
+	ToolsPanelHeaderProps,
+} from '../types';
+
+const DefaultControlsGroup = ( props: ToolsPanelControlsGroupProps ) => {
+	const { items, onClose, toggleItem } = props;
```

Comment:
You should be able to keep using the previous, more compact syntax here:

```suggestion
const DefaultControlsGroup = ( { items, onClose, toggleItem }: ToolsPanelControlsGroupProps ) => {
```